src/axolotl/utils/data.py

```diff
@@ -160,8 +161,26 @@ def load_tokenized_prepared_datasets(
                         split=None,
                     )
                 elif local_path.is_file():
+                    ds_type = "json"
+                    if d.ds_type:
+                        ds_type = d.ds_type
+                    elif d.data_files and (
+                        (
+                            isinstance(d.data_files, list)
+                            and ".parquet" in d.data_files[0]
+                        )
+                        or (
+                            isinstance(d.data_files, str) and ".parquet" in d.data_files
+                        )
+                    ):
+                        ds_type = "parquet"
+                    elif d.data_files and (
+                        (isinstance(d.data_files, list) and ".arrow" in d.data_files[0])
+                        or (isinstance(d.data_files, str) and ".arrow" in d.data_files)
+                    ):
```

Comment:
maybe can be refactored to inside one if

```py
                    elif d.data_files:
                        if (
                         (
                             isinstance(d.data_files, list)
                             and ".parquet" in d.data_files[0]
                         )
                         or (
                             isinstance(d.data_files, str) and ".parquet" in d.data_files
                         )
                         ):
                             ds_type = "parquet"
                         elif  (
                             (isinstance(d.data_files, list) and ".arrow" in d.data_files[0])
                             or (isinstance(d.data_files, str) and ".arrow" in d.data_files)
                         ):
                             ds_type = "arrow"
```

Reply:
haha yea. Would be good to add these examples to the Readme. One for parquet and arrow.